content/blog/2022-06-29-june-community-gems.md

```diff
@@ -0,0 +1,215 @@
+added data. If you're working with remotes using the `--to-remote` option, you
+can skip the local cache entirely and move the file contents directly to your
+remote storage.
+
+## [How can I connect Iterative Studio to a remote repo on a private network, like a GitLab server?](https://discord.com/channels/485586884165107732/563406153334128681/981543978644172830)
+
+Good question about [Iterative Studio](https://studio.iterative.ai/) from
+@LilDataScientist!
+
+This is something that our users asked quite a bit, so we wrote up a whole guide
+about
+[custom GitLab server connections](https://dvc.org/doc/studio/user-guide/connect-custom-gitlab-server).
+It's a quick walkthrough of how to set up the permissions you'll need and
+connecting your team to Studio.
+
+You can find lots of great guides and explanations about everything Studio in
+the [User Guide](https://dvc.org/doc/studio/user-guide) section of the docs!
+
+## [How does `dvc get-url` interact with the cache compared to `dvc import-url`?](https://discord.com/channels/485586884165107732/563406153334128681/981862313076346920)
+
+This is an awesome question from @Gema Parreno!
+
+When you run `dvc get-url`, it downloads the file/directory to your local file
+system. It's _not_ tracking the downloaded data with a `.dvc` file. It's just
+pulling that data from some source to your file system. If you want to download
+a file or directory without needing a DVC project, you can use the `dvc get-url`
+command.
+
+On the other hand, when you run `dvc import-url`, the local `cache` folder
+inside of `.dvc` will be updated. This is similar to running `dvc get-url` and
+`dvc add` together.
+
+There is one more option to bypass the local cache and transfer data directly to
+your remote storage using `dvc import-url <url> --to-remote`. This doesn't
+download anything to your local cache so it's another way to transfer data
+between remotes.
+
+## [If an image is present in different directories in different projects, will the shared cache store them both as one hash or will their different paths mean the same image appears twice in the cache?](https://discord.com/channels/485586884165107732/563406153334128681/984408209387298837)
+
+Great question about the cache from @paulwrightkcl!
+
+DVC will index the whole directory, but there will only be one hash per file. So
+the same image will only appear once in the cache. What _will_ be duplicated in
+the cache is the `.dir` hash that DVC uses internally as the directory tree
+representation.
+
+In summary, the image file is only stored in the shared cache once unless it's
+modified in one of the directories.
+
+## [Is it possible to limit which columns show for experiments in the metrics table?](https://discord.com/channels/485586884165107732/563406153334128681/985448515402616842)
+
+Nice question from @DylanTF!
+
+You can use `dvc exp --drop` (or `--keep`) to decide what to hide (or show). For
```

Comment:
Consider suggesting `--only-changed` here since it will automatically drop most columns that aren't useful.